server/common/auth/index.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const ActivityTracker = require('../../user/activity-tracker');
 const { auth: config = {} } = require('../../config');
 const { ExtractJwt, Strategy } = require('passport-jwt');
 const { User } = require('../database');
```

Comment:
Personally I would attach `ActivityTracker` to `User` class so you can destructure it here instead cope with `../../` paths...